source/_integrations/motion_blinds.markdown

```diff
@@ -30,3 +30,56 @@ Please note that "-" characters need to be included in the key when providing it
 <img src='/images/integrations/motion_blinds/Motion_App__get_key_1.jpg' />
 <img src='/images/integrations/motion_blinds/Motion_App__get_key_2.jpg' />
 </p>
+
+## Top Down Bottom Up (TDBU) blinds
+
+
+TDBU blinds consist of two bars controlled by two motors designated by Top and Bottom with fabric in between.
+The Top and Bottom can move independent of eachother to cover different parts of the window.
+Controlling the two bars can be done through three different entities that will be created: Top, Bottom and Combined.
+
+### Top entity
+
+- 'Up/Open' will move the Top bar to the top of the window (absolute position 100)
+- 'Down/Close' will move the Top bar to the position of the Bottom bar therefore making the part of the window that is covered as small as possible, but the two bars will be at the position of the Bottom bar (not at the top of the window).
+- 'Position' is the relative position in which the Top bar can move, so from the top of the window (100) to the position of the Bottom bar (0), note that the position will therefore change if the Bottom bar is moved, since the space in which the Top bar is allowed to move changes.
+- 'Absolute position' is the position of the Top bar with respect to the window, so 0 = bottom of the window and 100 = top of the window. Note that not all absolute positions are reachable at all moments due to the Bottom bar.
+- 'Width' is the percentage of the window covered by fabric (the space between the Top and Bottom bars).
```

Comment:
```suggestion
- 'Up/Open' will move the Top bar to the top of the window (absolute position 100).
- 'Down/Close' will move the Top bar to the position of the Bottom bar, therefore, making the part of the window that is covered as small as possible, but the two bars will be at the position of the Bottom bar (not at the top of the window).
- 'Position' is the relative position in which the Top bar can move, so from the top of the window (100) to the position of the Bottom bar (0), note that the position will therefore change if the Bottom bar is moved, since the space in which the Top bar is allowed to move changes.
- 'Absolute position' is the position of the Top bar with respect to the window, so 0 = bottom of the window and 100 = top of the window. Note that not all absolute positions are reachable at all moments due to the Bottom bar.
- 'Width' is the percentage of the window covered by fabric (the space between the Top and Bottom bars).
```